docs/libcudacxx/extended_api/memory.rst

```diff
@@ -45,11 +45,6 @@ Memory
      - CCCL 3.1.0
      - CUDA 13.1
 
-   * - :ref:`is_aligned <libcudacxx-extended-api-memory-is_aligned>`
-     - Check if a pointer is aligned
-     - CCCL 3.1.0
-     - CUDA 13.1
-
```

Comment:
I believe we should rather document `is_sufficiently_aligned` than just dropping `is_aligned`